selfdrive/car/volkswagen/mqbcan.py

```diff
@@ -40,8 +40,8 @@ def create_lka_hud_control(packer, bus, ldw_stock_values, enabled, steering_pres
     ]}
 
   values.update({
-    "LDW_Status_LED_gelb": 1 if enabled and steering_pressed else 0,
-    "LDW_Status_LED_gruen": 1 if enabled and not steering_pressed else 0,
+    "LDW_Status_LED_gelb": 1 if latActive and steering_pressed else 0,
+    "LDW_Status_LED_gruen": 1 if latActive and not steering_pressed else 0,
```

Comment:
enabled should be a superset of active, you mean openpilot can *not* be controlling steer while showing the icon (steer fault for example)?

Reply:
yes, i suppose thats a better description.
having it changed to active will also properly show when openpilot is at a standstill

i just tested it this morning, while at standstill the LKAS light turns off as openpilot is not active-laterally (expected behavior)